### PR TITLE
CONTRAST-31192 update .NET Framework docs 

### PR DESCRIPTION
**Pull request by DanFiedler**
- Change profiler chaining note to use the common config key rather than the legacy key
- Remove information changes that are no longer made by the .NET Framework installer
- Replace legacy configuration examples with common config examples throughout .NET Framework docs

### CODE REVIEW

File: content/installation/net/usage/NetApplicationPoolFiltering.md [hunk 1]
```diff
@@ -58,22 +58,26 @@ Use **Contrast .NET Logs**:
 
 ## Blacklist an Application Pool
 
-To disable the agent for a specific application, populate `ProcessBlacklist` with the appropriate application pool in *C:\Program Files\Contrast\dotnet\DotnetAgentService.exe.config*:
+To disable the agent for a specific application, populate `agent.dotnet.app_pool_blacklist` with the appropriate application pool in *C:\ProgramData\Contrast\dotnet\contrast_security.yaml*:
 
 ```
 <!--Comma-separated list of application pools ignored by Contrast-->
```
Comment (Silvenga) [inline content/installation/net/usage/NetApplicationPoolFiltering.md]: Maybe switch to yaml comment style here?

File: content/installation/net/usage/NetApplicationPoolFiltering.md [hunk 2]
```diff
@@ -58,22 +58,26 @@ Use **Contrast .NET Logs**:
...
 ## Whitelist an Application Pool 
 
-If you need to only enable the agent for specific applications hosted by IIS, configure a `ProcessWhitelist` to only analyze certain application pools. If an application pool is whitelisted, the agent analyzes the matching pools. There should be no performance impact for any other applications.
+If you need to only enable the agent for specific applications hosted by IIS, configure `agent.dotnet.app_pool_whitelist` to only analyze certain application pools. If an application pool is whitelisted, the agent analyzes the matching pools. There should be no performance impact for any other applications.
 
-To enable the agent for only specific application pools, populate `ProcessWhitelist` with the appropriate application pool in *C:\Program Files\Contrast\dotnet\DotnetAgentService.exe.config*:
+To enable the agent for only specific application pools, populate `agent.dotnet.app_pool_whitelist` with the appropriate application pool in *C:\ProgramData\Contrast\dotnet\contrast_security.yaml*:
 
 ```
 <!--Comma-separated list of application pools exclusively profiled by Contrast-->
```
Comment (Silvenga) [inline content/installation/net/usage/NetApplicationPoolFiltering.md]: Same as above.

File: content/troubleshooting/net/GettingNetAgentLogs.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ In rare scenarios, bad instrumentation causes a web server process to crash or a
 
 ## Agent Logs Directory
 
-The .NET agent logs information to the *Contrast\dotnet\LOGS* directory within *C:\ProgramData\Contrast\dotnet\LOGS*, the Windows 2008/2012 *ProgramData* directory. Depending on the setup of the Windows profile and folder view settings, the directories may be hidden. If so, paste the paths into the Windows Explorer location; you may need to replace the drive letter *C* with *D*.
+The .NET agent logs information to the *LOGS* directory within *C:\ProgramData\Contrast\dotnet\*, the Windows 2008/2012/2016 *ProgramData* directory. Depending on the setup of the Windows profile and folder view settings, the directories may be hidden. If so, paste the paths into the Windows Explorer location; you may need to replace the drive letter *C* with *D*.
```
Comment (Silvenga) [inline content/troubleshooting/net/GettingNetAgentLogs.md]: We specify the C:\ProgramData\Contrast\dotnet\ then talk about `ProgramData` directory?

File: content/troubleshooting/net/GettingNetAgentLogs.md [hunk 4]
```diff
@@ -68,10 +68,16 @@ Complete the following steps to gather information to send to Contrast.
   * [Stop the .NET agent service](http://127.0.0.1:9000/installation-netusage.html#usage).
   * Enable additional logging. 
      * ** Start > Notepad > Right click > Run as Administrator**
-     * ** File > Open >** *C:\Program Files\Contrast\dotnet\DotnetAgentService.exe.config*
-     * Change `ShouldLogMethodSignatures` to `true`.
-     * Uncomment the entry for `LogLevel`. 
-     * Check that the specified `LogLevel` value is `trace`.
+     * ** File > Open >** *C:\ProgramData\Contrast\dotnet\contrast_security.yaml*
```
Comment (Silvenga) [inline content/troubleshooting/net/GettingNetAgentLogs.md]: Do we need to edit this as admin anymore? I believe `C:\ProgramData\Contrast\dotnet` inherits from `C:\` which doesn't require admin, unless the installer modifies something.

File: content/installation/net/usage/NetApplicationPoolFiltering.md (hunk 1 above)
Comment (DanFiedler) [reply content/installation/net/usage/NetApplicationPoolFiltering.md]: Good idea. Switched.

File: content/troubleshooting/net/GettingNetAgentLogs.md (hunk 3 above)
Comment (DanFiedler) [reply content/troubleshooting/net/GettingNetAgentLogs.md]: Good idea. Switched.

File: content/troubleshooting/net/GettingNetAgentLogs.md (hunk 4 above)
Comment (DanFiedler) [reply content/troubleshooting/net/GettingNetAgentLogs.md]: removed the 'start as admin' text.

File: content/installation/net/usage/NetApplicationPoolFiltering.md (hunk 2 above)
Comment (DanFiedler) [reply content/installation/net/usage/NetApplicationPoolFiltering.md]: Good idea. Switched.

**Comment by kaitlynmortimer** [comment]
We need to make sure that @scorlett has these updates to the Troubleshooting docs.

**Comment by scorlett** [comment]
@marklacasse can you take a look at the updates to the [getting agent logs](https://github.com/Contrast-Security-OSS/docs/pull/632/files/b8fdaf4df89aaf8e32db8964fe42cb2573074e6c#diff-2647393f1683ce78a7ed5bb669bb4603) doc and update our KB article accordingly? Thank you!

**Pull request merged**